datahub-graphql-core/src/main/resources/app.graphql

```diff
@@ -456,6 +456,10 @@ type FeatureFlagsConfig {
   Whether we should show CTAs in the UI related to moving to Managed DataHub by Acryl.
   """
   showAcrylInfo: Boolean!
+  """
+  Whether ERModelRelation Tables Feature should be shown.
+  """
+  ermodelrelationFeatureEnabled: Boolean!
```

Comment:
let's rename and update casing to be consistent with application.yml: `erModelRelationshipFeatureEnabled`